sdk/turing/generated/model/id_object1.py

```diff
@@ -0,0 +1,179 @@
+"""
+    Turing Minimal Openapi Spec for SDK
+
+    No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)  # noqa: E501
+
+    The version of the OpenAPI document: 0.0.1
+    Generated by: https://openapi-generator.tech
+"""
+
+
+import re  # noqa: F401
+import sys  # noqa: F401
+
+from turing.generated.model_utils import (  # noqa: F401
+    ApiTypeError,
+    ModelComposed,
+    ModelNormal,
+    ModelSimple,
+    cached_property,
+    change_keys_js_to_python,
+    convert_js_args_to_python_args,
+    date,
+    datetime,
+    file_type,
+    none_type,
+    validate_get_composed_info,
+)
+
+
+class IdObject1(ModelSimple):
```

Comment:
Can we get rid of the repeated `IdObject` definitions in the jobs.yaml / router.yaml files and directly reference the one in common.yaml? So we wont have multiple such classes with the numeric suffixes.

Reply:
I've actually tried doing that but it seems like it triggers a bug in the OpenAPI specs bundling, whereby it creates a reference for the `IdObject` in `openapi.bundle.yaml` to `'#/components/schemas/~1projects~1{project_id}~1routers~1{router_id}~1undeploy'`. This then causes the Python OpenAPI autogenerated classes to import from a module titled `projects_project_id_routers_router_id`, which doesn't exist. I'm not sure if this sounds similar to anything you're encountered before though.

Reply:
Oops okay I figured that that might not have been the right way to reference it; lemme find out if there's a neater way to do this

Reply:
Okay I've taken a closer look at this, apparently according the [OpenAPI 3.0 standard](https://swagger.io/docs/specification/components/) (see the section on 'Externally Defined Components'), all external file paths are only used in references made in locally defined components. This is so that the latter act as 'local aliases' for the entire file (defining request bodies, responses, etc.). 

It seems like the OpenAPI generator also sticks to that convention and fails to generate the proper references if we try to make external references directly in other parts of the file (in this case it's the response definition for deleting routers) without first creating a local alias (a local component with the external reference). 

On the other hand if we were to try defining the `IdObject` in `router.yaml` (or `jobs.yaml`) with an external reference to the _other_ component instead, such that the definitions are linked up in a chain, i.e.:

In `router.yaml`:
```yaml
...
components:
  schemas:
    IdObject:
      $ref: "jobs.yaml#/components/schemas/IdObject"
...
```

In `jobs.yaml`:
```yaml
...
components:
  schemas:
    IdObject:
      $ref: "common.yaml#/components/schemas/IdObject"
...
```

This doesn't work either and still ends up generating 2 different `IdObject` classes. A similar issue with OpenAPI generator has been reported [here](https://github.com/OpenAPITools/openapi-generator/issues/2701), and it seems like it's still open from 2019 till today... :/ 

TL;DR: So all in all, I think we can't just erase the 2 `IdObject` definitions away from `router.yaml` or `jobs.yaml` :(

Reply:
I see.. Thanks for investigating this, @deadlycoconuts!

What if we use `RouterId` and `JobId` in the respective `routers.yaml` and `jobs.yaml` and files and no longer name the local component `IdObject` (only the item in `common.yaml` will be called `IdObject`) ?

Reply:
That works, but we'll have 3 classes instead of 2, but I guess that's way more elegant than having `IdObject` and `IdObject1` 😅 Thanks for the awesome solution; I'll put that in and merge the changes once I test it out again with the SDK.

Reply:
Cool! Thanks :)